telemetry/definitions/commonDefinitions.json

```diff
@@ -1423,6 +1435,11 @@
             "type": "string",
             "description": "User locale. Examples: en-US, en-GB, etc."
         },
+        {
+            "name": "manifestVersion",
```

Comment:
we also have `buildSystemVersion`. can we name this `version` so that we don't need specific "fooVersion" fields for each topic